### PR TITLE
create support page

### PR DESCRIPTION
**Pull request by wslack**
Created for https://github.com/18F/federalist-docs/issues/195

### CODE REVIEW

File: pages/using-federalist/support.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Support
+permalink: /pages/using-federalist/support/
+---
+# Getting Support
+
+After we execute an Inter-Agency Agreement (IAA), and you are a live customer, we will onboard you into our support process. Our goal is to provide you with support responsees within one business day though we aspire to be faster in our responses. As a reminder, Federalist agreements do not include labor hours, but we always want to make sure the platform is working and responsive to your needs as we consider new features.
```
Comment (jseppi) [inline pages/using-federalist/support.md]: `responsees` -> `responses`

File: pages/using-federalist/support.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+In general, when helping out with support requests, it is always helpful for users to include:
+
+- Full repository URL (like https://github.com/organization/repository)
+- The custom domain being used, if any
+- The name of the branch that is experiencing problems
+- Full build logs for the failing build
+
+When these points of information are not included, we often end up having to track them down, which slows down the debugging process. We may also ask you to to add Federalist staff to your repo with `write` privilages so that we can more quickly debug.
```
Comment (jseppi) [inline pages/using-federalist/support.md]: `privilages` -> `privileges`

File: _data/navigation.yml [hunk 3]
```diff
@@ -21,6 +21,8 @@ sidenav:
   - text: Using Federalist
     href: /pages/using-federalist/
     subfolderitems:
+      - text: Builds and Previews
```
Comment (jseppi) [inline _data/navigation.yml]: I think we want these in sentence case (ie `Builds and previews`), right?

File: _data/navigation.yml [hunk 4]
```diff
@@ -31,10 +33,10 @@ sidenav:
         href: /pages/using-federalist/launch-checklist/
       - text: Search on Federalist sites
         href: /pages/using-federalist/search/
-      - text: Viewing site previews
-        href: /pages/using-federalist/previews/
-      - text: Templates
+      - text: Site Templates
```
Comment (jseppi) [inline _data/navigation.yml]: I think we want these in sentence case (ie `Site templates`), right?

Comment (wslack) [reply _data/navigation.yml]: oh right, I caught myself last time with these

**Pull request merged**